packages/common/src/resourceLabel.ts

```diff
@@ -0,0 +1,11 @@
+export type ResourceLabel = `${string}__${string}`;
+
+export function resourceLabel({
```

Comment:
thoughts on `getResourceLabel` or `resourceToLabel`?

Reply:
Definitely prefer one of them

Reply:
Going with `resourceToLabel` for consistency with `resourceToHex`